packages/api-client/src/api/serializers/product.ts

```diff
@@ -237,7 +237,7 @@ const addHostToImage = (image, config: ApiConfig) => ({
     styles: image.attributes?.styles ? image.attributes.styles.map((style) => ({
       width: style.width,
       height: style.height,
-      url: (config.assetsUrl || config.backendUrl) + style.url
+      url: `${style.url.includes('http') ? '' : (config.assetsUrl || config.backendUrl)}${style.url}`
```

Comment:
Conceptually looks good. Could we maybe extract the `style.url.includes('http') ? '' : (config.assetsUrl || config.backendUrl)` part to a variable? It's not very clear just looking at it right now :)